apps/studio/components/interfaces/Settings/Database/DiskSizeConfiguration.tsx

```diff
@@ -61,133 +59,115 @@ const DiskSizeConfiguration = ({ disabled = false }: DiskSizeConfigurationProps)
-                        </Button>
-                      </div>
-                    </div>
-
-                    <div className="col-span-8">
-                      <Alert_Shadcn_>
-                        <Info size={16} />
-                        <AlertTitle_Shadcn_>Importing a lot of data?</AlertTitle_Shadcn_>
-                        <AlertDescription_Shadcn_>
-                          <Markdown
-                            className="max-w-full"
-                            content={`
-We auto-scale your disk as you need more storage, but can only do this once every 6 hours.
-If you upload more than 1.5x the current size of your storage, your database will go
-into read-only mode. If you know how big your database is going to be, you can
-manually increase the size here.
+        <>
+          <Admonition
+            type="default"
+            title="Importing a lot of data?"
+            description={
+              <Markdown
+                content={`
+We auto-scale your disk as you need more storage, but can only do this once every 6 hours. If you upload more than 1.5x the current size of your storage, your database will go into read-only mode. If you know how big your database is going to be, you can manually increase the size here.
 
 Read more about [disk management](https://supabase.com/docs/guides/platform/database-size#disk-management) and how to [free up storage space](https://supabase.com/docs/guides/platform/database-size#vacuum-operations).
 `}
-                          />
-                        </AlertDescription_Shadcn_>
-                      </Alert_Shadcn_>
+              />
+            }
+          />
+          <Card>
+            <CardHeader>Disk Size Configuration</CardHeader>
+
+            <CardContent className="flex justify-between items-center">
+              <FormLayout
+                layout="flex-row-reverse"
+                className="w-full"
+                label="Current Disk Storage"
+                description={
+                  <div className="flex text-lg mt-2 gap-6">
+                    <div className="flex flex-col items-center gap-1">
+                      <h5>Space used</h5>
+                      {formatBytes(databaseSizeBytesUsed, 2, 'GB')}
+                    </div>
+                    <div className="flex flex-col items-center gap-1">
+                      <h5>Total size</h5>
+                      {currentDiskSize} GB
                     </div>
                   </div>
-                </div>
-              </div>
-            </Panel.Content>
-          </Panel>
-        </div>
+                }
+              >
+                <Button asChild type="default" iconRight={<ExternalLink size={14} />}>
+                  <Link href={`/project/${projectRef}/reports/database#database-size-report`}>
+                    View detailed summary
+                  </Link>
+                </Button>
+              </FormLayout>
+            </CardContent>
```

Comment:
layout here is a bit different from before so that it better matches the content of the rest of the page